site/app/entities/forum/Thread.php

```diff
@@ -136,10 +136,14 @@ public function isPinned(): bool {
         return $this->pinned_expiration > new DateTime("now");
     }
 
-    public function IsPinnedExpiring(): bool {
+    public function isPinnedExpiring(): bool {
         return $this->pinned_expiration <= (new DateTime("now"))->add(DateInterval::createFromDateString("7 days"));
     }
 
+    public function isAnnounced(): bool {
+        return !is_null($this->announced);
```

Comment:
Minor stylistic comment: I personally think it's cleaner to directly compare to `null` instead of using `is_null()`.  You could consider applying the same throughout this PR.

Reply:
I prefer `is_null` generally. If a language is bothering to provide this library function, my preference is to use it and not have to worry about any hackery regarding direct comparison, even if they should be equivalent.